src/nostr-follow-button/nostr-follow-button.ts

```
@@ -0,0 +1,176 @@
+              if (signedUser) {
+                await signedUser.follow(userToFollow);
+              }
+
+              this.isFollowed = true;
+            }
+          }
+        } catch (err) {
+          this.isError = true;
+
+          const error = err as Error;
+          if (error.message && error.message.includes('NIP-07')) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)